#Работа ГБО. 0 - Выкл, 1 - Вкл
int8 Status